packages/mui-system/src/cssVars/createCssVarsProvider.d.ts

```diff
@@ -63,15 +63,50 @@ export default function createCssVarsProvider<
         >
       >;
   };
-  defaultColorScheme: DesignSystemColorScheme;
+  defaultColorScheme:
+    | DesignSystemColorScheme
+    | { day: DesignSystemColorScheme; night: DesignSystemColorScheme };
```

Comment:
```suggestion
    | { light: DesignSystemColorScheme; dark: DesignSystemColorScheme };
```
?

Reply:
Sure, I will change to `light` and `dark`.